examples/ct/handlers_test.go

```diff
@@ -346,8 +357,8 @@ func TestAddChain(t *testing.T) {
 			if got, want := ct.Version(resp.SCTVersion), ct.V1; got != want {
 				t.Errorf("resp.SCTVersion=%v; want %v", got, want)
 			}
-			if got, want := hex.EncodeToString(resp.ID), ctMockLogID; got != want {
```

Comment:
I suspect it was done this way to produce more readable error messages. I don't have a strong opinion on this though.